### PR TITLE
Add missing class for dropUp to work

### LINKED ISSUE #125: DropUp is not working with multiple triggerChars
As in the topic, dropUp is not working with multiple triggerChars. Here is my mention config:
`public mentionConfig = {
    mentions: [
      {
        items: ['Jan', 'Adam', 'Basia', 'Barbara'],
        triggerChar: '#'
      },
      {
        items: ['Jan', 'Adam', 'Basia', 'Barbara'],
        triggerChar: '@'
      }
    ],
    dropUp: true
  };`
Putting dropUp inside mentions didn't change anything. Any idea how to fix this? I can try do it on my own, but do you have any preferences @dmacfarlane ?

### PR DESCRIPTION
**Pull request by pgedzba**
Fixes #125 by adding missing style. Please check if the solution suits you @dmacfarlane

### CODE REVIEW

File: projects/angular-mentions/src/lib/mention-list.component.scss [hunk 1]
```diff
@@ -54,4 +54,9 @@
 [hidden] {
   display: none;
 }
-  
\ No newline at end of file
+
+.mention-dropdown-menu {
```
Comment (dmacfarlane) [inline projects/angular-mentions/src/lib/mention-list.component.scss]: Rename to `.mention-dropup` and remove the `!important` properties.

File: projects/angular-mentions/src/lib/mention-list.component.ts [hunk 2]
```diff
@@ -13,12 +13,12 @@ import { getCaretCoordinates } from './caret-coords';
...
 @Component({
   selector: 'mention-list',
-  styleUrls: ['./mention-list.component.css'],
+  styleUrls: ['./mention-list.component.scss'],
   template: `
     <ng-template #defaultItemTemplate let-item="item">
       {{item[labelKey]}}
     </ng-template>
-    <ul #list [hidden]="hidden" class="dropdown-menu scrollable-menu" [class.mention-menu]="!styleOff">
+    <ul #list [hidden]="hidden" class="dropdown-menu scrollable-menu" [class.mention-menu]="!styleOff" [class.mention-dropdown-menu]="!styleOff && dropUp">
```
Comment (dmacfarlane) [inline projects/angular-mentions/src/lib/mention-list.component.ts]: Add the new attribute on the next line to reduce line length.

**Comment by pgedzba** [comment]
Should be good now

**Comment by dmacfarlane** [comment]
Can you remove the `!important` properties? They shouldn't be required.

**Comment by pgedzba** [comment]
Sorry, forgot about this

**Pull request merged**